### PR TITLE
extra types: fix C type in plot time

### LINKED ISSUE #105: cannot use _Ctype_long(p.S)  as type _Ctype_longlong
Hello, 

While trying to build the example using the latest stable version of Go and the latest version of cimgui-go the compilation fails with:

`C:\Users\ArtoS\go\pkg\mod\github.com\!allen!dang\cimgui-go@v0.0.0-20230123025400-8ac6282740cd\extra_types.go:205:25: cannot use _Ctype_long(p.S) (value of type _Ctype_long) as type _Ctype_longlong in struct literal`

In the end, I had to replace the code in the extra_types.go (on line 205) with:

`return C.ImPlotTime{S: C.longlong(p.S), Us: C.int(p.Us)}`

OS: Windows
Compiler: gcc version 12.2.0 (Rev6, Built by MSYS2 project) 64 bit

Let me know if you need more information.

### PR DESCRIPTION
**Pull request by gucio321**
fix #105

### CODE REVIEW

File: extra_types.h [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 #ifdef __WIN32
 typedef unsigned long long xlong;
+typedef long long xlong;
```
Comment (nullptr-ref) [inline extra_types.h:5]: This should also be xxlong

**Review by nullptr-ref — CHANGES_REQUESTED** [review]

File: extra_types.h (hunk 1 above)
Comment (gucio321) [reply extra_types.h:5]: ops, right :smile:
thank you

File: extra_types.h [hunk 2]
```diff
@@ -5,5 +5,5 @@ typedef unsigned long long xulong;
 typedef long long xlong;
 #else
 typedef unsigned long xulong;
```
Comment (nullptr-ref) [inline extra_types.h:7]: This doesn't seem right, let me check. Think this will break a whole lot of stuff.

**Review by nullptr-ref — CHANGES_REQUESTED** [review]
As suspected, when renaming xlong to xulong it broke other entries of "xlong" in the following 2 files:

input_text.go
cimgui_funcs.go

You have to also replace them there. I'll attach the already replaced files.

**Review by nullptr-ref — APPROVED** [review]
All good now :)

**Review by nullptr-ref — APPROVED** [review]

**Review by AllenDang — APPROVED** [review]
LGTM

**Pull request merged**